fix-whitespace.yaml

```diff
@@ -0,0 +1,50 @@
+# fix-whitespace determines the set of directories first and
+# then the set of filenames inside those directories.
+#
+# The extended glob pattern can be used to specify file/direcotory names.
+# For details, see http://hackage.haskell.org/package/filemanip-0.3.6.3/docs/System-FilePath-GlobPattern.html
+#
+# By default, fix-whitespace checks every directory under the current working directory
+#
+#
+# Every directory matched by excluded-dirs is excluded unless it is included by included-dirs.
+included-dirs:
+  - src/full/Agda/Compiler/MAlonzo # Without this line it will be excluded.
+
+excluded-dirs:
+  - "**MAlonzo" # matches every MAlonzo in any directory including the src/full/Agda/Compiler/MAlonzo
+  - "**dist*"   # matches every dist* in any directory
+  - .stack-work
+  - _darcs
+  - .git
+  - std-lib
+  - test/bugs
```

Comment:
`test/bugs` no longer exists.
There is `test/Bugs`, but maybe it need not be excluded any more.

Reply:
Deleted!